src/components/plots/areaPlot.ts

```diff
@@ -6,6 +6,7 @@ export module Plots {
    * An AreaPlot draws a filled region (area) between the plot's projected "y" and projected "y0" values.
    */
   export class Area<X> extends Line<X> {
+    public static PLOT_CLASS = "area";
```

Comment:
We should use a consistent name between this and the `Plots.Line` version of this concept. Something like `LINE_PATH_CLASS` and `AREA_PATH_CLASS`?